sqrl-calcite/src/main/java/com/datasqrl/plan/validate/ScriptPlanner.java

```diff
@@ -155,35 +155,29 @@ public Void visit(SqrlCreateDefinition statement, Void context) {
     NamePath namePath = NamePath.of(statement.getName().names.toArray(String[]::new));
     String logId = statement.getName().names.get(0);
 
-    Map map = new HashMap();
-    map.put("primary-key", List.of("_uuid"));
-    map.put("timestamp-type", "LOG_TIME");
-    map.put("timestamp-name", "event_time");
-    map.put("name", logId);
-
-    Optional<TableConfig> sink = connectorFactoryFactory.create(Type.LOG, "kafka")
-        .map(t->t.createSourceAndSink(new ConnectorFactoryContext(logId, map)));
-
-    if (sink.isEmpty()) {
-      addError(ErrorCode.CANNOT_RESOLVE_TABLESINK, statement, "Cannot resolve log: %s",
-          logId);
-      return null;
-    }
-
-    TableConfig tableConfig = sink.get();
-    this.framework.getSchema().add(new ResolvedImport(logId, tableConfig));
     TypeFactory typeFactory = TypeFactory.getTypeFactory();
     RelDataTypeBuilder fieldBuilder = CalciteUtil.getRelTypeBuilder(typeFactory);
     fieldBuilder.add(Name.system("_uuid"), typeFactory.createSqlType(SqlTypeName.VARCHAR));
-//    fieldBuilder.add(Name.system("event_time"), typeFactory.createSqlType(SqlTypeName.TIMESTAMP_WITH_LOCAL_TIME_ZONE, 3));
+    // TODO: that's an ugly hack, postgres-log requires it as a field, however kafka requires it as metadata
```

Comment:
I didn't have a satisfactory answer to this, and I also felt that it might be better to address it in a follow-up discussion. I considered the possibility that we shouldn't dynamically add fields to a table that was defined using the "create table" command, but rather let the user specify them. However, I'm not certain if the "create table" command has a metadata syntax. Probably not, but wanted to think loud.

Reply:
Time in streaming systems is really complicated and it'll be easier for someone to use if we can manage it for them, much in the same way that we abstract away the nuances of timestamp management when writing queries.

Reply:
On it.